website/docs/r/en_subscription_email.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: 'Event Notifications'
+layout: 'ibm'
+page_title: 'IBM : ibm_en_subscription_email'
+description: |-
+  Manages Event Notifications SMS subscription.
+---
+
+# ibm_en_subscription_email
+
+Create, update, or delete a Email subscription by using IBM Cloud™ Event Notifications.
+
+## Example usage for Email Subscription Creation
+
+```terraform
+resource "ibm_en_subscription_email" "email_subscription" {
```

Comment:
Can you format the example in all docs to have proper indentation

Reply:
Updated all indentations.